modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastViewModel.kt

```diff
@@ -400,14 +403,18 @@ class PodcastViewModel
 
     fun play(bookmark: Bookmark) {
         val bookmarkEpisode = (uiState.value as? UiState.Loaded)?.episodes?.firstOrNull { it.uuid == bookmark.episodeUuid }
+        var shouldLoadOrSwitchEpisode = false
         bookmarkEpisode?.let {
-            val shouldPlayEpisode = !playbackManager.isPlaying() ||
+            shouldLoadOrSwitchEpisode = !playbackManager.isPlaying() ||
                 playbackManager.getCurrentEpisode()?.uuid != bookmarkEpisode.uuid
-            if (shouldPlayEpisode) {
+            if (shouldLoadOrSwitchEpisode) {
                 playbackManager.playNow(it, sourceView = SourceView.PODCAST_SCREEN)
             }
         }
-        playbackManager.seekToTimeMs(bookmark.timeSecs * 1000)
+        launch {
+            delay(if (shouldLoadOrSwitchEpisode) DELAY_IN_MS else 0) // Allow to load or switch episode
```

Comment:
This is the same as above, is it possible to set the episode playback position at the start?